### ISSUE TITLE
This package does not work with Node 10.x lambda runtime

### ISSUE THREAD

**Issue by jderose9** [Nike-Inc/lambda-logger-node#6]
This works for me in node 8 but when I switch to the node 10 lambda runtime I get this error:

````
2019-10-16T21:00:49.641Z undefined ERROR Uncaught Exception
{
    "errorType": "AssertionError [ERR_ASSERTION]",
    "errorMessage": "Input A expected to strictly equal input B:\n+ expected - actual\n\n- 1\n+ 0",
    "code": "ERR_ASSERTION",
    "stack": [
        "AssertionError [ERR_ASSERTION]: Input A expected to strictly equal input B:",
        "+ expected - actual",
        "",
        "- 1",
        "+ 0",
        "    at clearLambdaExceptionHandlers (/var/task/submit.js:3531:10)",
        "    at registerErrorHandlers (/var/task/submit.js:3498:3)",
        "    at Logger (/var/task/submit.js:3270:5)",
        "    at Module.<anonymous> (/var/task/submit.js:6653:14)",
        "    at __webpack_require__ (/var/task/submit.js:20:30)",
        "    at /var/task/submit.js:84:18",
        "    at Object.<anonymous> (/var/task/submit.js:87:10)",
        "    at Module._compile (internal/modules/cjs/loader.js:778:30)",
        "    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)",
        "    at Module.load (internal/modules/cjs/loader.js:653:32)"
    ],
    "generatedMessage": true,
    "name": "AssertionError [ERR_ASSERTION]",
    "actual": 1,
    "expected": 0,
    "operator": "strictEqual"
}
````

I believe it is due to these lines:
````
const assert = require('assert')
  // Taken from: https://gist.github.com/twolfson/855a823cfbd62d4c7405a38105c23fd3
  // DEV: AWS Lambda's `uncaughtException` handler logs `err.stack` and exits forcefully
  //   uncaughtException listeners = [function (err) { console.error(err.stack); process.exit(1); }]
  //   We remove it so we can catch async errors and report them to Rollbar
  assert.strictEqual(process.listeners('uncaughtException').length, 1)
  assert.strictEqual(process.listeners('unhandledRejection').length, 0)
  process.removeAllListeners('uncaughtException')
  hasAlreadyClearedLambdaHandlers = true
````

**Comment by scionaltera** [comment]
I'm seeing the same thing. `process.listeners('unhandledRejection')` equals 1 in the Node 10 runtime, so it blows up.

**Comment by kyeotic** [comment]
fix published under `3.1.2`
